### PR TITLE
Update gcc.py

### PR DESCRIPTION
**Pull request by JensTimmerman**
a recent llvm breaks the comparisonprocess between stage 2 and 3 for some reason. This should fix it, see https://github.com/Homebrew/homebrew/issues/38501

### CODE REVIEW

File: easybuild/easyblocks/g/gcc.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #
 # EasyBuild is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation v2.
+# the Free Software Foundation v2.bo
```
Comment (boegel) [inline easybuild/easyblocks/g/gcc.py]: hmm?

File: easybuild/easyblocks/g/gcc.py [hunk 2]
```diff
@@ -410,6 +410,10 @@ def build_step(self):
 
             # enable bootstrapping for self-containment
             configopts += " --enable-bootstrap "
+            # Use 'bootstrap-debug' build configuration to force stripping of object
+            # files prior to comparison during bootstrap (broken by Xcode 6.3).
+            # see https://github.com/Homebrew/homebrew/issues/38501
+            configopts += "--with-build-config=bootstrap-debug "
```
Comment (boegel) [inline easybuild/easyblocks/g/gcc.py]: at best, we should only do this when the OS is detected as being OS X (via `get_os_type()` from `easybuild.tools.systemtools`)

it's not clear to me what other implications this extra configopts would have

**Comment by boegel** [comment]
@JensTimmerman: care to mention why you're closing this?

File: easybuild/easyblocks/g/gcc.py [hunk 3]
```diff
@@ -410,6 +410,10 @@ def build_step(self):
 
             # enable bootstrapping for self-containment
             configopts += " --enable-bootstrap "
+            # Use 'bootstrap-debug' build configuration to force stripping of object
+            # files prior to comparison during bootstrap (broken by Xcode 6.3).
+            # see https://github.com/Homebrew/homebrew/issues/38501
```
Comment (boegel) [inline easybuild/easyblocks/g/gcc.py]: should also mention https://gcc.gnu.org/bugzilla/show_bug.cgi?id=65733

**Comment by JensTimmerman** [comment]
didn't completely fix it after all.

**Pull request closed without merging**